### ISSUE TITLE
USA McCormack tree missing

### ISSUE THREAD

**Issue by jimbo0810** [TNODevs/TNO-Bug-Tracker#3315]
### Operating System

windows 11 home

### HOI4 version

current version

### TNO version

current version

### Do you have other mods in use?

instant naval invasion, instant paratrooper drop, 2wrw mod, alantropia

### Were you using Steam?

Yes

### Which major expansions do you NOT have?

I have all

### Describe the bug

McCormack tree is completely missing 

### Steps to reproduce / Cause of the issue

once JFK is assassinated McCormack tree does not appear

### Expected behaviour

McCormack tree should be there

### Screenshots

![image_2024-06-20_211047841](https://github.com/TNODevs/TNO-Bug-Tracker/assets/138622718/6784cc7a-af5d-49c5-8900-698a7d4a2fcf)
![Screenshot 2024-06-20 210811](https://github.com/TNODevs/TNO-Bug-Tracker/assets/138622718/ba82233a-1c17-4128-9208-bac762617548)

**Comment by Marok132** [comment]
It's a save corruption. Start a new game and turn off the submods untill they get updates

**Comment by jimbo0810** [comment]
all submods are up to date

**Comment by Jack121Q** [comment]
Yeah, your save likely got corrupted. This works fine in-game. Sorry
